test/ds/test_dynamic_shape_models.py

```diff
@@ -155,6 +155,35 @@ def create_dynamic_test_data(self,
     y_test_nonzero_dev = torch.nonzero(y_test_xla.int()).float().squeeze()
     return x_test_nonzero_dev, y_test_nonzero_dev
 
+  def test_roialign(self):
```

Comment:
what's this test trying to test? running `roi_align_torchvision` will not crash?

Reply:
yes, that's the first step. By running this in our CI, we can make sure all other changes (pytorch, xla, runtime) won't break it (it won't crash.)

Correctness and number of compilation test will come next.